client/components/button/style.scss

```diff
@@ -76,6 +76,14 @@ button {
 			top: 4px;
 			margin-top: -8px;
 		}
+		// Make the left margin of the small plus icon visually less huge
+		.gridicons-plus-small {
```

Comment:
`:only-child` won't work because there's a `noscript` tag inside that button. What about using `:last-of-type`?